src/components/GovBanner/GovBanner.tsx

```diff
@@ -5,15 +5,117 @@ import classnames from 'classnames'
 import flagImg from 'uswds/src/img/us_flag_small.png'
 import dotGovIcon from 'uswds/src/img/icon-dot-gov.svg'
 import httpsIcon from 'uswds/src/img/icon-https.svg'
+import lockIcon from 'uswds/src/img/lock.svg'
+
+type Language = 'english' | 'spanish'
+
+type TLD = '.gov' | '.mil'
+
+interface GovBannerCopy {
+  header: string
+  headerAction: string
+  tldSectionHeader: string
+  tldSectionContent: JSX.Element
+  httpsSectionHeader: string
+  httpsSectionContent: JSX.Element
+}
+
+const getCopy = (language: Language, tld: TLD): GovBannerCopy => {
+  const lock = (
+    <span className="icon-lock">
+      <img src={lockIcon} className="usa-banner__lock-image" alt="lock" />
+    </span>
+  )
+
+  switch (language) {
+    case 'english':
+      return {
+        header: 'An official website of the United States government',
+        headerAction: 'Here’s how you know',
+        tldSectionHeader: `Offical websites use ${tld}`,
```

Comment:
typo: `Offical > Official`

Reply:
Good eye! 1378092

Reply:
Added snapshot tests in d8b3a91

I hadn't actually used those before, so this was a good learning experience. Please verify what I ended up doing is in line with convention for Snapshot tests at your convenience 😄